html/index.html

```diff
@@ -1,81 +1,334 @@
-                      </a>
-                  </li>
+                <ul class="social-media" id="social">
+                    <li>
+                        <a href="https://twitter.com/Mbonabucya12">
+                            <img src="../Image/Vector-twitter.svg" alt="Twitter icon" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/james-mbonabucya-88666910b/">
+                            <img src="../Image/vector-linkedin.svg" alt="Linkedin icon" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.gmail.com">
+                            <img src="../Image/Vector-Medium.svg" alt="Medium icon" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/mbonabucya">
+                            <img src="../Image/Vector-github.svg" alt="Github icon" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/mbonabucya">
+                            <img src="../Image/Vector-peace.svg" alt=" peace icon" />
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works-container">
+            <div class="tonic-project">
+                <div class="tonic">
+                    <div class="blog-background-Image">
+                    </div>
+                    <h1>
+                        Tonic
+                    </h1>
+                    <div class="tonic-header">
+                        <ul class="canopy-list">
+                            <li class="canopies">
+                                <p >CANOPY</p>
+                            </li>
+                            <li class="back">
+                                <p >Back End Dev</p>
+                            </li>
+                            <li class="back">
+                                <p>2015</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="tonic-body">
+                        <p>
+                          A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                </div> 
+                <div class="work-category">
+                    <ul class="work-category-language">
+                        <li>
+                            <p>Html</p>
+                           
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>javascript</p>
+                        </li>
```

Comment:
- [ ] Kindly make sure for each card in the **works section** their content in styling have correct values as the ones provided from Figma, such as:
- **font-weight**, **font-size**, **line-height** on texts
- Each is supposed to have a border. Kindly add it and use the right color, size, and style.
- Use the **right spacing** between cards (e.g: **margins: top - right - left - bottom **
## Figma design  
![card](https://user-images.githubusercontent.com/83042742/170831241-1d8bf27d-2d52-443a-a3af-36ba13f1b56f.png) 
## Your design 
 ![s_card](https://user-images.githubusercontent.com/83042742/170831292-5e7520c0-6a9b-49f5-a0e0-cce7bf7c73d1.png)